src/benchmarks/micro/corefx/System.Text.Json/Serializer/WriteJson.cs

```diff
@@ -54,6 +54,10 @@ public async Task SerializeToStream()
             await JsonSerializer.SerializeAsync(_memoryStream, _value);
         }
 
+        [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+        [Benchmark]
+        public string SerializeObjectProperty() => JsonSerializer.Serialize(new { Prop = (object)_value });
```

Comment:
Would it be possible to move the creation of the `new { Prop = (object)_value }` out of the benchmark?

In the current version, the benchmark includes the cost of creating and serializing the object, while if possible it should be the cost of serialization only.

Reply:
Good point, fixed.